lib/policy.py

```diff
@@ -1334,6 +1349,7 @@ class VarType(object):
   HOP_LIMIT = 47
   LOG_NAME = 48
   FLEXIBLE_MATCH_RANGE = 49
+  PAN_APPLICATION = 50
```

Comment:
Can we change this to 54? I have some internal stuff I will be pushing out soon and it will conflict.